bool collect_flag
---
string voice_filename
